### DISCUSSION TITLE
local settings  file path

### DISCUSSION THREAD

**Discussion by LXD312569496** [russhwolf/multiplatform-settings#115] [Q&A]
Where is the local settings file path in windows?
I want to look and open this file。

**Comment by russhwolf** [comment]
It saves to the registry. You can look at it in regedit. If you do `WindowsSettings("foo")` it will put your keys under `HKEY_CURRENT_USER\foo`. If you do `WindowsSettings.Factory("foo").create("bar")` it will put your keys under `HKEY_CURRENT_USER\SOFTWARE\foo\bar`.

**Comment by DestBro** [comment]
What about JvmPropertiesSettings ?

**Reply by russhwolf** [reply]
On construction, you can pass an `onModify` lambda in which you can update your properties file. Otherwise the `Properties` class only lives in-memory on the JVM.
